### PR TITLE
List,snackbar and grids

### PR DESCRIPTION
**Pull request by Hardiegogo**
Added : 

- Lists
    - Spaced list
    - Stacked list
- Snackbar
- Grids
    - Two columns grid
    - Three columns grid
    - Grid auto fit
    
    Preview link available below.

### CODE REVIEW

**Review by ashishnnnnn — APPROVED** [review]
Code seems good to merge :+1:

File: CSS_files/components/grids/grids.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.grid-2{
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 1rem;
```
Comment (Prabhav-Sharma) [inline CSS_files/components/grids/grids.css]: You can use global variables for properties like margin, gap, and padding, like
--space-xsmall:0.5rem

**Comment by Hardiegogo** [comment]
> Code seems good to merge 👍

Thank you.

File: CSS_files/components/grids/grids.css (hunk 1 above)
Comment (Hardiegogo) [reply CSS_files/components/grids/grids.css]: Ohh yes, I've created utility classes for margin and padding. As I was not using the gap property much, I didn't create a utility class for it. 
Thanks for the feedback, I will use global variables for such cases from now on.

Comment (Hardiegogo) [reply CSS_files/components/grids/grids.css]: > You can use global variables for properties like margin, gap, and padding, like --space-xsmall:0.5rem

I have made the necessary changes, Thank you.

**Pull request merged**